range.go

```diff
@@ -182,6 +188,34 @@ func (r Range) Flip(ranges []Range) []Range {
 	return r.flipValidRanges(rngs)
 }
 
+func (r Range) MarshalJSON() ([]byte, error) {
+	data := marshalTime{
+		StartTime: r.st,
+		EndTime:   r.End(),
+	}
+
+	jsonData, err := json.Marshal(data)
+	if err != nil {
+		fmt.Println("Error marshaling JSON:", err)
```

Comment:
why do force print needed here. Could you provide more context please where do you need use stdout f.e. in standard libraries

Reply:
I agree with that point. An average consumer of the library don't want an arbitrary error to appear in stdout, but rather to print the error, that has been returned by the function on his own. Also, I would recommend you to wrap the error with  the message, like "unmarshal json data", e.g.:
```go
return nil, fmt.Errorf("unmarshal start and date: %w", err)
```